scripts/build_nginx_image.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
```

Comment:
It’s here initially because it uses some of the tooling in the `scripts` folder. Can be moved, just a little more fiddly.